### DISCUSSION TITLE
Show/Hide Direct Links

### DISCUSSION THREAD

**Discussion by dycoit** [KelvinTegelaar/CIPP#2051] [General]
Not sure if this is a bug or not.  On the "Tenants Administrator - Administrator - Tenants" page.  The page used to default to show the individual links to all the portals.  Since an update in the last 1 month at the most, it's started to default to hiding all this and just giving the menu 3 dots on the right.  Clicking the "Show direct links" at the top brings it back but is there a way to have it default so users don't need to click this every time or is this a bug?

**Comment by KelvinTegelaar** [comment]
Not a bug, you've set this in your settings, closing because its a support question

**Comment by dycoit** [comment]
Can I ask where in settings?  Only thing I can find that sounds like it may be it is "Show compressed tenant list" in user settings, but disabling/enabling this doesn't change anything.

**Comment by KelvinTegelaar** [comment]
No you cant, there is no free support. Only sponsors get support :)
